saleor/graphql/views.py

```diff
@@ -273,7 +288,11 @@ def parse_query(
             return None, ExecutionResult(errors=[e], invalid=True)
 
     def execute_graphql_request(self, request: HttpRequest, data: dict):
-        with tracer.start_as_current_span("graphql_query") as span:
+        with (
+            tracer.start_as_current_span("graphql_query", service_scope=True) as span,
```

Comment:
Are we sure all data in `graphql_query` should be exposed to `SERVICE` tracker?

Reply:
Yes, we agreed that all the attributes are suitable for the service scope